.travis.yml

```diff
@@ -1,40 +1,133 @@
-language: python
+conditions: v1
 
-sudo: false
+dist: xenial
+language: python
+python:
+- "3.7"
```

Comment:
Use single quotes here, and no quotes for the `pypy...` entries.

Reply:
I know, but I wanted to be more consistent. + I'm don't remember how well Ruby's yaml parser consumes single quotes